pennylane/pulse/parametrized_evolution.py

```diff
@@ -502,3 +504,16 @@ def fun(y, t):
         elif not self.hyperparameters["return_intermediate"]:
             mat = mat[-1]
         return qml.math.expand_matrix(mat, wires=self.wires, wire_order=wire_order)
+
+
+@functions.bind_new_parameters.register
+def _bind_new_parameters_parametrized_evol(op: ParametrizedEvolution, params: Sequence[TensorLike]):
```

Comment:
Why not put this inside `ops/functions/bind_new_parameters.py`? Is it because that would create a circular dependency?

Reply:
Yep, because `ParametrizedEvolution` isn't available top-level.